Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Commands/Settings/RunCommandSettings.cs

```diff
@@ -61,4 +62,24 @@ public sealed class RunCommandSettings : CommandSettings
     [CommandOption("--sqs-eventsource-config <CONFIG>")]
     [Description("The configuration for the SQS event source. The format of the config is a comma delimited key pairs. For example \"QueueUrl=<queue-url>,FunctionName=<function-name>,VisibilityTimeout=100\". Possible keys are: BatchSize, DisableMessageDelete, FunctionName, LambdaRuntimeApi, Profile, QueueUrl, Region, VisibilityTimeout")]
     public string? SQSEventSourceConfig { get; set; }
+
+    /// <summary>
+    /// The absolute path used to save global settings and saved requests. You will need to specify a path in order to enable saving global settings and requests.
+    /// </summary>
+    [CommandOption("--config-storage-path <CONFIG-STORAGE-PATH>")]
+    [Description("The absolute path used to save global settings and saved requests. You will need to specify a path in order to enable saving global settings and requests.")]
+    public string? ConfigStoragePath { get; set; }
+
+    /// <summary>
+    /// Validate that <see cref="ConfigStoragePath"/> is an absolute path.
+    /// </summary>
+    public override ValidationResult Validate()
+    {
+        if (string.IsNullOrEmpty(ConfigStoragePath))
+            return ValidationResult.Success();
+
+        return !Path.IsPathFullyQualified(ConfigStoragePath)
+            ? ValidationResult.Error("'Config storage path' must be an absolute path.")

```

Comment:
Why does the path have to be fully qualified? That seems really unintuitive and limiting.

Reply:
because as Garrett mentioned, using . starts from the nuget package and not the current folder.

Reply:
Sorry that was a leftover artifact when I wanted to recreate what @GarrettBeatty saw. Ignore it.